modules/ROOT/pages/standalone-mule-setup.adoc

```diff
@@ -15,8 +15,8 @@ Anypoint Visualizer supports the following Mule versions for standalone Mule dep
 == Install Agent
 
 To support Anypoint Visualizer for standalone Mule deployments, install the xref:monitoring::am-installing.adoc[Anypoint Monitoring agent].
-Anypoint Visualizer also requires an additional header injection for standalone Mule deployments.
-However, due to a potential performance impact for Mule 4, header injection is disabled by default. 
+Anypoint Visualizer requires an additional header injection for standalone Mule deployments to appear properly with their correct connections.
+Due to a potential performance impact, however, the header injection is disabled by default for Mule 4 apps. 
```

Comment:
Hmm...this sounds kind of important. Perhaps the best way to treat a default disablement the must be enabled for Visualizer is by rewording a bit:

"Although header injection is disabled by default for Mule 4 apps, Anypoint Visualizer requires you to provide an additional header injection for standalone Mule deployments so that the deployments display the correct connections."